pkg/grafana/grafana.go

```diff
@@ -196,8 +195,8 @@ func ConfigureDefaultDatasources(ctx context.Context, grafanaAPI *client.Grafana
 
 	for _, datasource := range datasourcesToUpdate {
 		logger.Info("updating datasource", "datasource", datasource.Name)
-		_, err := grafanaAPI.Datasources.UpdateDataSourceByID(
-			strconv.FormatInt(datasource.ID, 10),
+		_, err := grafanaAPI.Datasources.UpdateDataSourceByUID(
+			datasource.UID,
```

Comment:
Maybe we do not need the datasource ID in the organization status anymore 🤔

Reply:
Maybe, but that's a bigger change and I am not sure about the implications there.

Reply:
I don't think we should do it in that PR specifically :)